### DISCUSSION TITLE
different base url

### DISCUSSION THREAD

**Discussion by ah-rahimi** [def-studio/telegraph#132] [General]
@fabio-ivona 
I have self host in server 
And I use http://localhost:8000 instead of https://api.telegram.org
How I can changed it?

**Comment by fabio-ivona** [comment]
This is not how it works, that is the telegram api server, not yours

You need an https domain as well in order to use webhooks

**Reply by ah-rahimi** [reply]
no 
It is for download & upload to 2GB & for telegram 
Just need change base url & It works

**Reply by ah-rahimi** [reply]
If you set the base url in telegraph config,
It will be fixed

**Reply by ah-rahimi** [reply]
& can create method in send messages & documents &... About what sending base url 
This is perfect

**Reply by fabio-ivona** [reply]
@ahr1384 feel free to open a PR for that

**Reply by ah-rahimi** [reply]
@fabio-ivona I don't have enough time 
Is it possible for you to add this item?
I really need this

**Reply by fabio-ivona** [reply]
I'm out on vacation right now, I'm sorry. Will take a look when I'll be back

**Reply by ah-rahimi** [reply]
thanks
